package.json

```diff
@@ -18,7 +18,7 @@
     "@types/config": "3.3.4",
     "@types/jira-client": "^7.0.0",
     "applicationinsights": "^3.0.0",
-    "cacheables": "^2.0.0",
```

Comment:
this library is hard to use and I just couldn't get it to work when it wasn't returning an http promise, `cajache` seems to work fine